src/OpenLoco/Title.cpp

```diff
@@ -83,13 +163,73 @@ namespace OpenLoco::Title
         initialiseViewports();
         sub_4284C8();
         Gui::init();
-        sub_444357();
+        reset();
         Gfx::invalidateScreen();
         resetScreenAge();
 
         Audio::playTitleScreenMusic();
     }
 
+    // 0x00444387
+    void update()
+    {
+        if (!isTitleMode())
+            return;
+
+        resetScreenAge();
+
+        if (_waitCounter > 0)
+        {
+            _waitCounter -= 1;
+            return;
+        }
+
+        do
+        {
+            auto& command = *_sequenceIterator++;
+            std::visit(overloaded{
+                           [](WaitStep step) {
+                               _waitCounter = step.duration / 4;
+                           },
+                           [](ReloadStep step) {
+                               loadTitle();
+                               Gfx::invalidateScreen();
+                               resetScreenAge();
+                               addr<0x50C19A, uint16_t>() = 55000;
+                           },
+                           [](MoveStep step) {
+                               if (addr<0x00525E28, uint32_t>() & 1)
+                               {
+                                   auto pos = Map::map_pos(step) + Map::map_pos(16, 16);
```

Comment:
This doesn't seem that elegant, but I don't see how else to account for the + 1 on both coors.

Reply:
@ZehMatt How would you suggest I go about this using the fancy new vectors?

Reply:
Let's introduce a TileManager::toTileCentre utility function.